### PR TITLE
Wen branch

### PR DESCRIPTION
**Pull request by WenLuo-EASYGO**
Update_11.04.2023

### CODE REVIEW

File: GeothermalEnergy/_config.yml
Comment (tlmerbecks) [inline GeothermalEnergy/_config.yml:1]: I would prefer if we make no changes to the default set up for now, especially things like referencing style as everyone will have a different preferred style.

Please let me know (on Slack) if this commit is absolutely necessary and why

File: GeothermalEnergy/_toc.yml
Comment (tlmerbecks) [inline GeothermalEnergy/_toc.yml:1]: Same as above, I do not think that we should be editing these too much for now

File: GeothermalEnergy/bibliography.md
Comment (tlmerbecks) [inline GeothermalEnergy/bibliography.md:1]: I cannot really see what has changed... I added a bibliography file last time... are you sure you updated your project properly?

**Review by tlmerbecks — CHANGES_REQUESTED** [review]
Please have a look at my comments for the *.yml files and the bibliography file

File: GeothermalEnergy/bibliography.md
Comment (WenLuo-EASYGO) [reply GeothermalEnergy/bibliography.md:1]: Not very sure. I didn't see your bibliography.md file in the main branch. So I created a new one here.

File: GeothermalEnergy/_config.yml
Comment (WenLuo-EASYGO) [reply GeothermalEnergy/_config.yml:1]: thanks for your time. I am ok to keep them as it is.

**Pull request merged**